skimage/util/tests/test_arraypad.py

```diff
@@ -993,7 +993,7 @@ def test_check_kwarg_not_allowed(self):
 
     def test_mode_not_set(self):
         arr = np.arange(30).reshape(5, 6)
-        assert_raises(ValueError, pad, arr, 4)
+        assert_raises((ValueError, TypeError), pad, arr, 4)
```

Comment:
When I ran the function directly, it gave me a helpful message saying the `mode` is invalid behind a `ValueError` but the wrapper complains of a TypeError when the 3rd argument is missing. I think both errors are valid, and there's no particular need to enforce one over the other in the test suite.